pkg/traces/config.go

```diff
@@ -147,10 +148,16 @@ const (
 	protocolHTTP    = "http"
 )
 
-// DefaultRemoteWriteConfig holds the default setting	s for a PushConfig.
+const (
+	formatOtel   = "otlp"
```

Comment:
Nit: can we change it to `formatOTLP`?

```suggestion
	formatOTLP   = "otlp"
```